packages/models/src/step/service.ts

```diff
@@ -21,6 +24,17 @@ const isSameStep = <T extends {}>(
   return leftStep?.id === rightStep?.id;
 };
 
+/**
+ * Public copy of the lesson, accessible by non collaborators.
+ */
+const isStepPublicDocument = (step: Step) => !!step.docId;
```

Comment:
I don't think we need `docId` anymore. The way system is going to work looks a lot different than it used to. Doc id was helpful when everything revolved around lessons/template. Since this is not the case anymore I think we can drop it and find a better way for handling published subjects.